nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -1,24 +1,97 @@
 define([
-
+    'bluebird',
+    'common/ui',
+    'common/html',
+    'common/events'
 ], (
-
+    Promise,
+    UI,
+    html,
+    Events
 ) => {
     'use strict';
 
-    class ConfigureWidget {
-        constructor() {
-            this.container = null;
+    const div = html.tag('div'),
+        span = html.tag('span'),
+        form = html.tag('form');
+
+    function ConfigureWidget(options) {
+        const bus = options.bus;
+        let container = null,
+            ui = null;
+
+        /**
+         * args includes:
+         *  - node - the DOM node to act as this widget's container
+         *  - something something inputs and parameters
+         * @param {object} args
+         */
+        function start(args) {
+            return Promise.try(() => {
+                container = args.node;
+                ui = UI.make({
+                    node: container,
+                    bus: bus
+                });
+                const layout = renderLayout();
+                container.innerHTML = layout.content;
+                layout.events.attachEvents(container);
+            });
         }
 
-        start(args) {
-            this.container = args.node;
-            this.container.innerHTML = 'configure some things!';
+        function renderLayout() {
+            let events = Events.make(),
+                formContent = [
+                    ui.buildPanel({
+                        title: span([
+                            'File Paths',
+                            span({
+                                dataElement: 'advanced-hidden-message',
+                                style: {
```

Comment:
Do we want to remove the inline styling?

Reply:
Yup, I can do that.